source/renderer/app/i18n/locales/en-US.json

```diff
@@ -1147,6 +1147,15 @@
   "wallet.summary.transactionsList.syncingTransactionsMessage": "Your transaction history for this wallet is being synced with the blockchain.",
   "wallet.summary.transactionsList.todayLabel": "Today",
   "wallet.summary.transactionsList.yesterdayLabel": "Yesterday",
+  "wallet.token.picker.addButtonLabel": "Add",
+  "wallet.token.picker.allTokensLabel": "All tokens",
+  "wallet.token.picker.cancelButtonLabel": "Cancel",
+  "wallet.token.picker.checkAllLabel": "Select all",
+  "wallet.token.picker.checkMaxFirstLabel": "Select first {maxTokens}",
```

Comment:
word tokens is missing 
https://docs.google.com/document/d/1rAxEXGx2RV_FMoKwcZYh2HdbjEssXJf9H7DjGO5mWF4/edit